decidim/decidim
# Admin log filters

#### :tophat: What? Why?
Implements filters to the admin logs view as described at #9180.

**WIP** This is still partly work in progress as in the sprint review meeting we discussed today that the participatory space dropdown may be problematic in instances that have many spaces, such as Barcelona. We decided that we are going to implement an autocomplete field for the participatory space selection to tackle this problem.

#### :pushpin: Related Issues
- Fixes #9180

#### Testing
- Login as an admin
- Go to the admin panel
- Visit the admin logs view
- See the filter inputs at the top of the view

#### :clipboard: Checklist


### :camera: Screenshots

![Admin log filters](https://user-images.githubusercontent.com/864340/175035404-fb247c53-7fd7-498a-8dea-454469a5c844.png)
